### DISCUSSION TITLE
PrimeVue Components sizez

### DISCUSSION THREAD

**Discussion by Aizaz-Rehman** [primefaces/community#2164] [PrimeVue]
i Have install primevue as documented from https://primevue.org/vite/ and the size i am getting and they described are different 
For example:
**My PrimeVue button    <Button label="Normal" icon="pi pi-check" /> have the size as they have provided for large button in documentation
my small size button size is equal to the normal size button they provided in the documentation** 
![MyPrimeVue](https://github.com/primefaces/community/assets/85885814/acdcb8ba-3fd1-4e81-9dff-6cb2b258db5a)
![PrimeVueButtons](https://github.com/primefaces/community/assets/85885814/7c5f60af-66f1-4ca0-be4d-181a6a4bee88)

**Comment by bbensing** [comment]
I have the same problem here  :-(

**Comment by EthanVn** [comment]
because of the line-height Attribute
in official documentation, line-height of html is normal
in code demo, line-height of html is 1.5
I think this property should have a default value to ensure that the documentation and actual code display the same effect
